addons/website/views/snippets/snippets.xml

```diff
@@ -729,6 +732,13 @@
         <t t-set="with_shapes" t-value="True"/>
     </t>
 
+    <div data-selector=".s_cta_badge">
+        <t t-call="website.snippet_options_border_widgets">
+            <t t-set="so_rounded_no_dependencies" t-value="True"/>
+        </t>
+        <t t-call="website.snippet_options_shadow_widgets"/>
+    </div>
+
```

Comment:
Define snippet's specific options in its xml file, `addons/website/views/snippets/s_cta_badge.xml` ;)
Also, add a `data-js` attribute in order to trigger the functionality correctly, normally `data-js="Box"` suits the scenario 👍